edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_variables.scss

```diff
@@ -2,14 +2,19 @@
 
 //color
 $primary: #B70005;
+$primary-dark: #a52238;
```

Comment:
Use uppercase characters to define colour values. @kuipumu

Reply:
OK, I changed them because the edX stylelint configuration raised a warning on uppercase hex color values